vmdb/app/models/pg_lock.rb

```diff
@@ -0,0 +1,29 @@
+class PgLock < ActiveRecord::Base
```

Comment:
Would it make sense to name this VmdbDatabaseLock or something to match the other namings?